src/Libraries/CoreNodeModels/Input/IntegerSlider.cs

```diff
@@ -343,7 +343,7 @@ private void UpdateNodeInfo(string value)
             }
             else
             {
-                Info(Resources.IntegerSliderInfoMessage);
+                Info(Resources.IntegerSliderInfoMessage, true);
```

Comment:
I'm curious about this one... shouldn't this be cleared when the graph executes again as the value will be potentially different? Perhaps I don't understand when an info should be persistent or not.

Reply:
Because we are updating the slider value for user in this case to max value of Int64, I think it would be better if the info remains so they know where to come back and fix it.